app/assets/javascripts/avalon/media-player.es6

```diff
@@ -105,11 +101,15 @@ export default class MediaPlayer {
     
     
     playFromHash() {
+        if (this.timer) {
+            clearTimeout(this.timer)
+            this.timer = null
+        }
         var mediaFragment = this.getMediaFragment(window.location.hash)
         var mediaPlayer = document.getElementById('iiif-av-player')
         mediaPlayer.setCurrentTime(mediaFragment.start)
         mediaPlayer.play()
-        setTimeout(() => {
+        this.timer = setTimeout(() => {
```

Comment:
I'll be happy to remove this! @bkeese from the current example, what do you think would be useful? 
Like in the TOC, the current segment that is playing is highlighted? Do you know if medialement can highlight a particular segment in the video timeline/bar?